code/modules/grey_market/grey_market_item.dm

```diff
@@ -170,229 +164,278 @@ var/locations_calculated = 0
 	desc = "Used medical waste."
 	item = /obj/item/stack/sheet/animalhide/human
-	sps_chances = list(5, 10, 15)
 	stock_min = 10
 	stock_max = 20
 	cost_min = 75
 	cost_max = 100
 	display_chance = 70
 
-/datum/black_market_item/plants
-	category = "Seeds"
-
-/datum/black_market_item/plants/mushroommanspore
+/datum/grey_market_item/agriculture/mushroommanspore
 	name = "Packet of Walking Mushroom Seeds"
-	desc = "Sentient mushfriends for all your mushy needs"
+	desc = "Sentient mushfriends for all your mushy needs."
 	item = /obj/item/seeds/mushroommanspore
-	sps_chances = list(0, 10, 30)
-	delivery_available = list(0, 1, 1)
+	delivery_available = list(1, 1, 1)
 	stock_min = 3
 	stock_max = 3
 	cost_min = 50
 	cost_max = 100
-	display_chance = 99
-
-/datum/black_market_item/food
-	category = "Cooked Goods and Kitchen Items."
+	display_chance = 99	
 
-/datum/black_market_item/food/bearmeat
+/datum/grey_market_item/agriculture/bearmeat
 	name = "Bear Meat"
 	desc = "A slab of bear meat for the manliest men."
 	item = /obj/item/weapon/reagent_containers/food/snacks/meat/bearmeat
-	sps_chances = list(0, 5, 10)
 	stock_min = 3
 	stock_max = 9
 	cost_min = 50
 	cost_max = 75
 	display_chance = 40
 
-/datum/black_market_item/food/xenomeat
+/datum/grey_market_item/agriculture/xenomeat
 	name = "Strange Meat"
 	desc = "An alien slab of meat."
 	item = /obj/item/weapon/reagent_containers/food/snacks/meat/xenomeat
-	sps_chances = list(0, 5, 10)
 	stock_min = 3
 	stock_max = 9
 	cost_min = 50
 	cost_max = 75
 	display_chance = 40
 
-/datum/black_market_item/food/human
+/datum/grey_market_item/agriculture/humanmeat
 	name = "Human Meat"
 	desc = "Don't ask questions."
 	item = /obj/item/weapon/reagent_containers/food/snacks/meat/human
-	sps_chances = list(5, 10, 15)
 	stock_min = 9
 	stock_max = 18
 	cost_min = 100
 	cost_max = 150
 	display_chance = 69
 
-/datum/black_market_item/food/blackfoodcolor
+/datum/grey_market_item/agriculture/blackfoodcolor
 	name = "Black Food Color"
 	desc = "The namesake for this whole market. You should buy one for that reason alone."
 	item = /obj/item/weapon/reagent_containers/food/drinks/coloring
-	sps_chances = list(5, 25, 30)
 	delivery_available = list(0, 1, 1)
 	stock_min = 1
 	stock_max = 5
 	cost_min = 150
 	cost_max = 225
 	display_chance = 90
 
-/datum/black_market_item/tool
-	category = "Utility Items"
+/datum/grey_market_item/tool
+	category = "Tools and Utility"
 
-/datum/black_market_item/tool/stethoscope
+/datum/grey_market_item/tool/stethoscope
 	name = "Stethoscope"
 	desc = "For medical use only, honest!"
 	item = /obj/item/clothing/accessory/stethoscope
-	sps_chances = list(5, 25, 30)
 	stock_min = 1
 	stock_max = 50
 	cost_min = 50
 	cost_max = 150
 	display_chance = 90
 
-/datum/black_market_item/toy
-	category = "Recreational and Novelty Items"
+/datum/grey_market_item/toy
+	category = "Recreational Goods"
 
-/datum/black_market_item/toy/dorkcube
+/datum/grey_market_item/toy/levitation
```

Comment:
This was hit in black market culling, so I dunno how people are gonna feel about it

Reply:
Removed

Reply:
The potion is still here.  Are you missing a commit?